### ISSUE TITLE
launch.json is not generated if Visual Studio solution folder is opened 

### ISSUE THREAD

**Issue by alexeychikk** [OmniSharp/omnisharp-vscode#3771]
## Issue Description ##
I created solution in Visual Studio 2019. Visual Studio 2019 really sucks so I decided to switch to VS Code. I installed all the necessary extensions and opened the solution's folder in VS Code. It seems that it scanned something but it didn't generate `launch.json` file so I could run my project.
Running `.NET: Generate Assets for Build and Debug` command creates only `tasks.json` and doesn't generate `launch.json`.

## Steps to Reproduce ##
Above.
## Expected Behavior ##
`launch.json` is generated 
## Actual Behavior ##
no `launch.json`
### OmniSharp log ###
<details>Starting OmniSharp server at 09.05.2020, 17:37:32
    Target: d:\git\Majiculate\Majiculate.sln

OmniSharp server started.
    Path: c:\Users\alexz\.vscode\extensions\ms-dotnettools.csharp-1.21.18\.omnisharp\1.35.1\OmniSharp.exe
    PID: 14844

[info]: OmniSharp.Stdio.Host
        Starting OmniSharp on Windows 6.2.9200.0 (x64)
[info]: OmniSharp.Services.DotNetCliService
        DotNetPath set to dotnet
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Located 3 MSBuild instance(s)
            1: Visual Studio Build Tools 2017 15.9.28307.1093 - "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin"
            2: Visual Studio Community 2019 16.5.30011.22 - "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin"
            3: StandAlone 16.4 - "c:\Users\alexz\.vscode\extensions\ms-dotnettools.csharp-1.21.18\.omnisharp\1.35.1\.msbuild\Current\Bin"
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Registered MSBuild instance: Visual Studio Community 2019 16.5.30011.22 - "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin"
[info]: OmniSharp.Cake.CakeProjectSystem
        Detecting Cake files in 'd:\git\Majiculate'.
[info]: OmniSharp.Cake.CakeProjectSystem
        Could not find any Cake files
[info]: OmniSharp.MSBuild.ProjectSystem
        Detecting projects in 'd:\git\Majiculate\Majiculate.sln'.
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'd:\git\Majiculate\Majiculate\Majiculate.csproj'
[info]: OmniSharp.Script.ScriptProjectSystem
        Detecting CSX files in 'd:\git\Majiculate'.
[info]: OmniSharp.Script.ScriptProjectSystem
        Could not find any CSX files
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.CSharpFormattingWorkspaceOptionsProvider, Order: 0
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: d:\git\Majiculate\Majiculate\Majiculate.csproj
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.RenameWorkspaceOptionsProvider, Order: 100
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.ImplementTypeWorkspaceOptionsProvider, Order: 110
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.BlockStructureWorkspaceOptionsProvider, Order: 140
[info]: OmniSharp.WorkspaceInitializer
        Configuration finished.
[info]: OmniSharp.Stdio.Host
        Omnisharp server running using Stdio at location 'd:\git\Majiculate' on host 14104.
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'd:\git\Majiculate\Majiculate\Majiculate.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'd:\git\Majiculate\Majiculate\Majiculate.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: Majiculate</details>

### C# log ###
<details>Post the output from Output-->C# here</details>

## Environment information ##

**VSCode version**: 1.45.0
**C# Extension**: 1.21.18

<details><summary>Dotnet Information</summary>
����� SDK ��� .NET Core (��ࠦ��騩 �� global.json):
 Version:   3.1.201
 Commit:    b1768b4ae7

�।� �믮������:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.1.201\

Host (useful for support):
  Version: 3.1.3
  Commit:  4a9f85e9f8

.NET Core SDKs installed:
  3.1.201 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|
|---|---|---|
|change-case|wmaurer|1.0.0|
|code-settings-sync|Shan|3.4.3|
|code-spell-checker|streetsidesoftware|1.8.0|
|color-info|bierner|0.5.1|
|csharp|ms-dotnettools|1.21.18|
|csharpextensions|jchannon|1.3.1|
|dracula-theme-from-intellij|trinm1709|0.3.0|
|git-graph|mhutchie|1.22.0|
|gitignore|codezombiech|0.6.0|
|gitlens|eamodio|10.2.1|
|intellij-idea-keybindings|k--kato|0.2.38|
|markdown-all-in-one|yzhang|2.8.0|
|material-icon-theme|PKief|4.1.0|
|mono-debug|ms-vscode|0.15.8|
|unsaved-files-vscode|wraith13|2.1.1|
|vscode-nuget-package-manager|jmrog|1.1.6|
|vscodeintellicode|VisualStudioExptTeam|1.2.7|;

</details>

**Comment by cartermp** [comment]
@alexeychikk Do you have a sharable codebase so that we can reproduce this? Thanks!

**Comment by cartermp** [comment]
Closing out as no additional info was added.

**Comment by ifrh** [comment]
May be I have an equal problem ... or just a Newbee blackout (I never have used C#): 
I tried on Windows 10 to compile in Visual Studio Code: 1.55.1  the project : https://github.com/timdams/pptx2tex
That project comes with no launch.json. 
That project has a "Microsoft Visual Studio Solution File, Format Version 12.00".
I installed "Open XML SDK 2.0" because it is mentioned at https://github.com/timdams/Pptx2Tex/blob/master/README.md as project depenency. 
I have installed on the system:

-  MS .NET Core SDK 3.1.407 (x64) and (x86) 
-  MS .NET Framework 4.5.2 Multi-Targeting Pack
-  MS .NET SDK 5.0.202 (x64) and (x86)

But VSCode do not create a launch.json after using "File - open Folder" 

And if I choose from "Run and Debug"  "create a launch.json file" that the result seems to be incomplete, 
no compilation is possible ...
